autoload/denops/api/vim.vim

```diff
@@ -22,9 +22,7 @@ endfunction
 function! denops#api#vim#batch(calls) abort
   let l:results = []
   try
-    for l:Call in a:calls
-      call add(l:results, call(l:Call[0], l:Call[1:]))
-    endfor
+    call map(a:calls, {_, v -> add(l:results, call(v[0], v[1:]))})
```

Comment:
Maybe using `mapnew()` instead `map()` and `add()` would be a bit more efficient.

Reply:
If an error occurs in `call()`, `mapnew()` does not save progress, so I decided to use `add()` instead.